<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DJ::View::MainDialog</class>
 <widget class="QDialog" name="DJ::View::MainDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>460</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>460</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>DOMjura</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>1: Source</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <property name="verticalSpacing">
         <number>0</number>
        </property>
        <item row="0" column="0">
         <widget class="QRadioButton" name="radioDOMjudge">
          <property name="text">
           <string>DOMjudge</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QComboBox" name="comboProtocol">
            <item>
             <property name="text">
              <string>http://</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>https://</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEditURL"/>
          </item>
          <item>
           <widget class="QLabel" name="labelDOMjudgePath">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="text">
             <string>/plugin/{scoreboard,event}.php</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="1" column="0">
         <widget class="QRadioButton" name="radioLocalXML">
          <property name="text">
           <string>Local XML files from directory</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLineEdit" name="lineEditLocalDir">
            <property name="enabled">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="buttonBrowseLocal">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Browse</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="2" column="1">
         <widget class="QLabel" name="labelDirInfo">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>(this directory should contain a &quot;scoreboard.xml&quot; and a &quot;event.xml&quot; file)</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QPushButton" name="buttonLoadData">
            <property name="text">
             <string>Load data</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="buttonSaveXMLData">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Save XML files</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="title">
      <string>2: Actions</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="2">
       <widget class="QPushButton" name="buttonViewStats">
        <property name="text">
         <string>View stats</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QPushButton" name="buttonStartResultsPresentation">
        <property name="text">
         <string>Start results presentation</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Branding image for results</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="lineEditBrandingImag"/>
      </item>
      <item row="2" column="2" rowspan="2">
       <widget class="QPushButton" name="buttonBrowseBrandingImage">
        <property name="text">
         <string>Browse</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>11</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="buttonQuit">
       <property name="text">
        <string>Quit</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonAbout">
       <property name="text">
        <string>About</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>radioDOMjudge</sender>
   <signal>toggled(bool)</signal>
   <receiver>comboProtocol</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>131</x>
     <y>47</y>
    </hint>
    <hint type="destinationlabel">
     <x>304</x>
     <y>46</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioDOMjudge</sender>
   <signal>toggled(bool)</signal>
   <receiver>lineEditURL</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>131</x>
     <y>47</y>
    </hint>
    <hint type="destinationlabel">
     <x>462</x>
     <y>45</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioDOMjudge</sender>
   <signal>toggled(bool)</signal>
   <receiver>labelDOMjudgePath</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>131</x>
     <y>47</y>
    </hint>
    <hint type="destinationlabel">
     <x>671</x>
     <y>45</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioLocalXML</sender>
   <signal>toggled(bool)</signal>
   <receiver>lineEditLocalDir</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>131</x>
     <y>88</y>
    </hint>
    <hint type="destinationlabel">
     <x>468</x>
     <y>85</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioLocalXML</sender>
   <signal>toggled(bool)</signal>
   <receiver>buttonBrowseLocal</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>131</x>
     <y>88</y>
    </hint>
    <hint type="destinationlabel">
     <x>729</x>
     <y>87</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonQuit</sender>
   <signal>clicked()</signal>
   <receiver>DJ::View::MainDialog</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>670</x>
     <y>401</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>214</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioLocalXML</sender>
   <signal>toggled(bool)</signal>
   <receiver>labelDirInfo</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>133</x>
     <y>90</y>
    </hint>
    <hint type="destinationlabel">
     <x>510</x>
     <y>124</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
